src/asteroid_tracker/asteroid_tracker.py

```diff
@@ -13,50 +16,122 @@
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
-
+format_Error = False
 
 API_URL = 'https://api.nasa.gov/neo/rest/v1/feed?start_date=%s&end_date=%s&api_key=DEMO_KEY'
 
 
 def main():
     """
-    Main function which parses input and calculates statistics.
+    Main function which takes input arguments and calls relevant functions.
     """
 
     start_date = None
     end_date = None
 
     # TODO: Task 1 - Gather user input
+    start_date, end_date = input(" ").split()
```

Comment:
The string value of `" "` does not mean anything here. It is supposed to be a message that displays before input e.g.
`start_date, end_date = input("Please enter two dates: ").split()`
Both dates are read in as a single string, and the `split` method separates them into two separate dates based on spaces.